prototypes/recipes/mukmoux/recipes-mukmoux.lua

```diff
@@ -213,11 +216,78 @@ RECIPE {
         {type = 'item', name = 'mukmoux', amount = 1, probability = 0.5},
     },
     --main_product = "mukmoux-mk02",
-    icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk02.png',
+    icons =
+        {
+            {icon = '__pyalienlifegraphics__/graphics/icons/evolution-mk02.png', icon_size = 64},
```

Comment:
do you have any convention about '' vs "" and general formating? I would like to adhere to your standards but it was that inconsistent from where I copied these lines